transforms3d/quaternions.py

```diff
@@ -82,16 +86,16 @@ def fillpositive(xyz, w2_thresh=None):
     # If necessary, guess precision of input
     if w2_thresh is None:
         try: # trap errors for non-array, integer array
-            w2_thresh = -np.finfo(xyz.dtype).eps * 3
+            w2_thresh = -np.finfo(xyz.dtype).eps
```

Comment:
Not sure about this one - the transforms3d code being more liberal in this case - is that bad?